transformer_lens/loading_from_pretrained.py

```diff
@@ -942,6 +942,7 @@ def get_pretrained_model_config(
     device: Optional[str] = None,
     n_devices: int = 1,
     default_prepend_bos: bool = True,
+    max_context_length: Optional[int] = None,
```

Comment:
I'm not positive if this is the correct default to use here.

My thinking is that "get_pretrained_model_config" implies that it will return the config mostly unaltered

Reply:
I would call it `override_default_context_length` maybe? That has the problem of sounding like a Boolean, but combined with the type being Optional[int] I think it's pretty obvious what's going on

Reply:
My reasoning for max vs. override was that max makes it clear that the context length will only be changed if its too big, whereas there is some ambiguity about whether override would always set it to a fixed value.

Reply:
perhaps `override_max_context_length` would actually be best

Reply:
`override_max_context_length` sounds good to me